students.md

```diff
@@ -6,3 +6,4 @@
 | Mark | mjw-18 | Way lab | Burgers |
 | Roshan | [@roshankern](https://github.com/roshankern) | Way lab | Hash Browns |
 | Jenna | [@jenna-tomkinson] (https://github.com/jenna-tomkinson) | [@wayscience] (https://github.com/WayScience) | Mac and Cheese |
+| Rose | [@rosedoss] (https://github.com/rosedoss) | Way Lab | Tacos |
```

Comment:
```suggestion
| Rose | [@rosedoss](https://github.com/rosedoss) | Way Lab | Tacos |
```

It looks like I missed this suggestion on @jenna-tomkinson's PR #7

There needs to be no space to enable the hyperlink!